cmd/agent/main.go

```diff
@@ -1,3 +1,107 @@
 package main
 
-func main() {}
+import (
+	"math/rand"
+	"net/http"
+	"runtime"
+	"strconv"
+	"time"
+)
+
+type MemStorage struct {
```

Comment:
CHANGE REQUEST: please move it to another package

Reply:
done